SPECS/kernel/config

```diff
@@ -1870,9 +1870,11 @@ CONFIG_PREVENT_FIRMWARE_BUILD=y
 # Firmware loader
 #
 CONFIG_FW_LOADER=y
+CONFIG_FW_LOADER_PAGED_BUF=y
```

Comment:
Can you add these config changes to the ARM config as well? SPECS/kernel/config_aarch64